Automattic/wp-calypso
# Change the order in locale switcher to match the landing pages

Related to #257-gh-Automattic/i18n-issues

## Proposed Changes

Change the order of the locales in the language switcher to make it more intuitive.

## Why are these changes being made?

The current list of locales is sorted by the locale code, which is not intuitive to our international users.
The rationale for the new ordering is: start with Latin-based ones sorted by locale code, followed by Cyrillic, Greek, RTL and CCJK locales, sorted by locale code within each group.

## Testing Instructions



On calypso.live or locale instance open /themes and /plugins URLs and confirm that the order of the locales in the language switcher generally follows the order from wordpress.com or wordpress.com/pricing, with some changes: the addition of Greek and Romanian and English being at the top in /themes and /plugins.

## Pre-merge Checklist
